### PR TITLE
UI redesign

### LINKED ISSUE #171: [Feature]: UI Redesign
### What feature?

Redesign the overall design. Make it ばえ！

### PR DESCRIPTION
**Pull request by siberowl**
## Which Issue was this PR made for?

- Closes #171

## What is within the scope with this PR

Redesigning the ui.

### CODE REVIEW

**Comment by siberowl** [comment]
image
image

File: client/src/pages/RoomPage.tsx [hunk 1]
```diff
@@ -105,6 +106,10 @@ function RoomPage({ name }: { name: string }): JSX.Element {
     const onKey = (e) => {
       if (e.key === "m") {
         toggleMute();
+      } else if (e.key === "s") {
+        setShowModal(true);
+      } else if (e.key === "L") {
+        history.go(0);
```
Comment (Njima1572) [inline client/src/pages/RoomPage.tsx:112]: Love the shortcut keys

**Review by Njima1572 — APPROVED** [review]
UI Seems way nicer.

There is a weird bug with toggling active, can be solved later with different issue once we know what's the problem.

File: client/src/pages/RoomPage.tsx [hunk 2]
```diff
@@ -105,6 +106,10 @@ function RoomPage({ name }: { name: string }): JSX.Element {
...
       } else if (e.code === "Space") {
```
Comment (Njima1572) [inline client/src/pages/RoomPage.tsx:113]: Space actually also triggers onClick when the button is focused, so might not be the good choice.
Change this to 's' for status and ',' for preferences?

**Review by Njima1572 — CHANGES_REQUESTED** [review]

**Review by Njima1572 — APPROVED** [review]
LGTM

**Pull request merged**